### PR TITLE
update armature

### PR DESCRIPTION
**Pull request by 2youyou2**
1.fixed movement event
2.update TEST_COLLIDER_DETECTOR

### CODE REVIEW

**Comment by minggo** [comment]
Please fix compiling error.

**Comment by 2youyou2** [comment]
This is a  jsb binding compiler error.
I have told Wen Hai.

File: scripting/javascript/bindings/generated [hunk 1]
```diff
@@ -1 +1 @@
-Subproject commit 88f9b35f3acef523f29171b56b276315b87be4c7
+Subproject commit f48957ca302e2f81c5d43cb162f916be17ff9416
```
Comment (minggo) [inline scripting/javascript/bindings/generated]: Please don't modify the submodule link.

File: tools/bindings-generator [hunk 2]
```diff
@@ -1 +1 @@
-Subproject commit 65664649d5a169f79ce026bba8edc4f9639d282e
+Subproject commit 2024736e2321063c027282283e9d7d014e93c774
```
Comment (minggo) [inline tools/bindings-generator]: Please don't modify the submodule link.

**Pull request merged**